src/types.ts

```diff
@@ -17,6 +17,7 @@ export interface Account {
   config: string;
   name: string;
   nostrPrivateKey?: string | null;
+  mnemonic?: string | null;
```

Comment:
changePassword is not re-encrypting the mnemnoic

Reply:
I've added that. :heavy_check_mark: